### ISSUE TITLE
Block "Power Saver" mode while video is playing

### ISSUE THREAD

**Issue by feross** [feross/webtorrent-app#106]
> The `powerSaveBlocker` module is used to block the system from entering low-power (sleep) mode and thus allowing the app to keep the system and screen active.

http://electron.atom.io/docs/v0.36.8/api/power-save-blocker/

**Comment by giuscri** [comment]
Where it's better to start the `powerSaveBlocker`? To me, it may make sense to put it [when the action `playPause` is dispatched](https://github.com/feross/webtorrent-app/blob/c176e3897ac70d55acd099a4f15464fb85e79ed7/renderer/index.js#L203-L206) via something like

```
if (action === 'playPause') {
  state.video.isPaused = !state.video.isPaused
  update()

  var id = state.powerSaveBlockerId
  if (id && powerSaveBlocker.isStarted(id)) {
    powerSaveBlocker.stop(id)
  } else {
    state.powerSaveBlockerId = powerSaveBlocker.start('prevent-display-sleep')
  }
}
```

*But*, recently a new action to be dispatched [has been added](https://github.com/feross/webtorrent-app/blob/c176e3897ac70d55acd099a4f15464fb85e79ed7/renderer/index.js#L199-L201): `pause`.

I can't grep any part of the codebase where this action is *emitted*, so ...why it has been added?, is that an error?, is it ok to delete it?

**Comment by Flet** [comment]
`pause` is emitted via IPC

setup here:
https://github.com/feross/webtorrent-app/blob/master/renderer/index.js#L226
```js
function setupIpc () {
  ipcRenderer.on('dispatch', function (e, action, ...args) {
    dispatch(action, ...args)
  })
```

and called when the window is "closed" on a mac:
https://github.com/feross/webtorrent-app/blob/master/main/windows.js#L53
```js
  win.on('close', function (e) {
    if (process.platform === 'darwin' && !isQuitting) {
      e.preventDefault()
      // sendSync() ensures that video will pause before window is hidden, at which point
      // the update() loop (which uses requestAnimationFrame) ceases to run
      win.sendSync('dispatch', 'pause')
      win.hide()
    }
  })
```

**Comment by giuscri** [comment]
To my understanding, this `pause` action is triggered only (on Mac's) as a cleanup when the application is closing.
Then, we don't need to add the `powerSaveBlocker` logic there. Correct?

**Comment by giuscri** [comment]
Adding logic for starting/stopping the `powerSaveBlocker` inside the handler of the `playPause` action introduces duplicate code, since we need to cover also when the player is *opened* for the first time.

I propose to modify the `autoplay` property of the `<video>`, and make `openPlayer` dispatch a `playPause` action instead.

By that, we could write the code for the `powerSaveBlocker` only in the handler of such action.

**Comment by ungoldman** [comment]
Should be solved by #140.
